src/features/defi/providers/idle/components/IdleManager/Overview/IdleOverview.tsx

```diff
@@ -46,6 +48,25 @@ const defaultMenu: DefiButtonProps[] = [
   },
 ]
 
+// Tried to get this using TS to be dynamic but can't seem to get it to work
+// Manually setting this to the tags we know of here.
+type Tags = 'Best Yield' | 'Junior Tranche' | 'Senior Tranche'
+
+const IdleTagDescriptions: Record<Tags, TagDescription> = {
```

Comment:
camelCase constant names.

```suggestion
const idleTagDescriptions: Record<Tags, TagDescription> = {
```